tests/misc-utils.h

```diff
@@ -41,6 +41,12 @@ object_array_new (void)
   return g_ptr_array_new_with_free_func (g_object_unref);
 }
 
+static inline gboolean
+flag_check (guint flags, guint flag)
+{
+  return (flags & flag) == flag;
```

Comment:
…but anyone reading the code has to go and look up what `flag_check()` does. Whatever.